VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -154,63 +159,89 @@ public void ConfigureServices(IServiceCollection services)
             // which saves you from doing the mapping in your authorization controller.
             services.Configure<IdentityOptions>(options =>
             {
-                options.ClaimsIdentity.UserNameClaimType = OpenIdConnectConstants.Claims.Name;
-                options.ClaimsIdentity.UserIdClaimType = OpenIdConnectConstants.Claims.Subject;
+                options.ClaimsIdentity.UserNameClaimType = OpenIdConnectConstants.Claims.Subject;
+                options.ClaimsIdentity.UserIdClaimType = OpenIdConnectConstants.Claims.Name;
                 options.ClaimsIdentity.RoleClaimType = OpenIdConnectConstants.Claims.Role;
             });
 
 
             // Register the OAuth2 validation handler.
-            services.AddAuthentication().AddOAuthValidation();
+            JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear();
```

Comment:
So I need to your comments about this configurations

Reply:
It is from example from [https://github.com/openiddict/openiddict-core/issues/577](https://github.com/openiddict/openiddict-core/issues/577)